pkg/cuectx/ctx.go

```diff
@@ -70,6 +70,7 @@ func LoadGrafanaInstancesWithThema(path string, cueFS fs.FS, lib thema.Library,
 
 	val := lib.Context().BuildInstance(inst)
 
+	opts = append(opts, thema.SkipBuggyChecks())
```

Comment:
Had to turn this on because of grafana/thema#68. Technically it's a sledgehammer because it affects ALL lineages we load, but because these checks in Thema only apply to lineages with >1 schema, it's fine as a temporary measure while i get that fixed.